XRTK-Core/Packages/com.xrtk.core/Inspectors/Profiles/MixedRealityBoundaryVisualizationProfileInspector.cs

```diff
@@ -35,31 +36,43 @@ public class MixedRealityBoundaryVisualizationProfileInspector : BaseMixedRealit
         private readonly GUIContent showContent = new GUIContent("Show");
         private readonly GUIContent scaleContent = new GUIContent("Scale");
         private readonly GUIContent materialContent = new GUIContent("Material");
+        private readonly GUIContent generalSettingsFoldoutHeader = new GUIContent("General Settings");
+        private readonly GUIContent floorSettingsFoldoutHeader = new GUIContent("Floor Settings");
+        private readonly GUIContent playAreaSettingsFoldoutHeader = new GUIContent("Play Area Settings");
+        private readonly GUIContent trackedAreaSettingsFoldoutHeader = new GUIContent("Tracked Area Settings");
+        private readonly GUIContent boundaryWallSettingsFoldoutHeader = new GUIContent("Boundary Wall Settings");
+        private readonly GUIContent boundaryCeilingSettingsFoldoutHeader = new GUIContent("Boundary Ceiling Settings");
 
         protected override void OnEnable()
         {
             base.OnEnable();
 
             boundaryHeight = serializedObject.FindProperty(nameof(boundaryHeight));
+            boundaryHeight.isExpanded = true;
 
             showFloor = serializedObject.FindProperty(nameof(showFloor));
+            showFloor.isExpanded = true;
             floorMaterial = serializedObject.FindProperty(nameof(floorMaterial));
             floorScale = serializedObject.FindProperty(nameof(floorScale));
             floorPhysicsLayer = serializedObject.FindProperty(nameof(floorPhysicsLayer));
 
             showPlayArea = serializedObject.FindProperty(nameof(showPlayArea));
+            showPlayArea.isExpanded = true;
             playAreaMaterial = serializedObject.FindProperty(nameof(playAreaMaterial));
             playAreaPhysicsLayer = serializedObject.FindProperty(nameof(playAreaPhysicsLayer));
 
             showTrackedArea = serializedObject.FindProperty(nameof(showTrackedArea));
+            showTrackedArea.isExpanded = true;
             trackedAreaMaterial = serializedObject.FindProperty(nameof(trackedAreaMaterial));
             trackedAreaPhysicsLayer = serializedObject.FindProperty(nameof(trackedAreaPhysicsLayer));
 
             showBoundaryWalls = serializedObject.FindProperty(nameof(showBoundaryWalls));
+            showBoundaryWalls.isExpanded = true;
             boundaryWallMaterial = serializedObject.FindProperty(nameof(boundaryWallMaterial));
             boundaryWallsPhysicsLayer = serializedObject.FindProperty(nameof(boundaryWallsPhysicsLayer));
 
             showBoundaryCeiling = serializedObject.FindProperty(nameof(showBoundaryCeiling));
+            showBoundaryCeiling.isExpanded = true;
```

Comment:
Not sure if we want to automatically expand all these things.

Reply:
I went for the "non-breaking change" way. Since they were not hidden initially.

Reply:
Should be okay I think.